docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/exitDisclaimer/exitDisclaimer.js

```diff
@@ -41,6 +41,9 @@ function _initialize(filterSetName) {
 	// move the feature card exit notification within the dom to come right after the image in the feature card to meet design request, WCMSFEQ-282
 	$('.feature-card a.icon-exit-notification').insertAfter('.feature-card a:not([href^="/"]):not([href*=".gov"]) div img');
 
+  // remove the exit notification from embedded cards which get their own external link icon
+  $('.cgdp-embed-card a.icon-exit-notification').remove();
+
```

Comment:
Can you do something in the legacyFilterSet to deal with not adding instead of removing?